test/libyul/YulOptimizerTestCommon.cpp

```diff
@@ -356,6 +357,11 @@ YulOptimizerTestCommon::YulOptimizerTestCommon(
 			FakeUnreachableGenerator fakeUnreachableGenerator;
 			fakeUnreachableGenerator(*m_ast);
 			StackLimitEvader::run(*m_context, *m_object, fakeUnreachableGenerator.fakeUnreachables);
+		}},
+		{"commonSwitchCasePrefixMover", [&] () {
+			disambiguate();
+                        FunctionHoister::run(*m_context, *m_ast);
```

Comment:
Formatting error: whitespace instead of tabs?

Reply:
Oh, sorry about that :-). I wonder why the style checker script did not complain about that. Anyways, fixed it!